src/DropDownMenu/DropDownMenu.spec.js

```diff
@@ -31,6 +32,43 @@ describe('<DropDownMenu />', () => {
     });
   });
 
+  describe('prop: disabled', () => {
+    it('disables the touchRipple effect if disableTouchRipple is undefined and disabled={true}', () => {
+      const wrapper = mountWithContext(
```

Comment:
I don't think that we need to use the mount API. The shallow API should be enough for all the new tests.